File: src/main/java/uk/co/threebugs/mochiwhattotrade3/setupgroups/SetupGroupsFile.java
package uk.co.threebugs.mochiwhattotrade3.setupgroups;

import lombok.Builder;
import lombok.Value;
import lombok.extern.jackson.Jacksonized;

import java.util.List;

@Jacksonized
@Value
@Builder
public class SetupGroupsFile {
    String name;
    String scriptsDirectory;
    List<SetupGroupFile> setupGroups;
}

File: backend/src/main/kotlin/uk/co/threebugs/darwinexclient/setupgroups/SetupGroupsFile.kt
package uk.co.threebugs.darwinexclient.setupgroups

import uk.co.threebugs.darwinexclient.setupgroup.*

data class SetupGroupsFile(
    val name: String,
    val scriptsDirectory: String,
    val setupGroups: List<SetupGroupFile>
)
